docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -306,3 +288,120 @@ Le code ci-dessus génère ce qui suit sur la page :
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### Structure du dossier
+
+Utilisez le composant `<FileTree>` pour afficher la structure d'un répertoire avec des icônes de fichiers et des sous-répertoires repliables.
+
+Spécifiez la structure de vos fichiers et répertoires avec une [liste Markdown non ordonnée](https://www.markdownguide.org/basic-syntax/#unordered-lists) dans `<FileTree>`.
+Créez un sous-répertoire en utilisant une liste imbriquée ou ajoutez un `/` à la fin d'un élément de liste pour le rendre comme un répertoire sans contenu spécifique.
```

Comment:
```suggestion
Créez un sous-répertoire en utilisant une liste imbriquée ou ajoutez un `/` à la fin d'un élément de liste pour l'afficher comme un répertoire sans contenu spécifique.
```

We tend to avoid translating `render` with `rendre` most of the time.